### PR TITLE
Run lint:fix

### PR DESCRIPTION
**Pull request by nick4598**

### CODE REVIEW

**Comment by nick4598** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/iTwin/imodel-transformer/152?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `federation-guid-optimization`
  * **PR #146** <a href="https://app.graphite.dev/github/pr/iTwin/imodel-transformer/146?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #150** <a href="https://app.graphite.dev/github/pr/iTwin/imodel-transformer/150?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #151** <a href="https://app.graphite.dev/github/pr/iTwin/imodel-transformer/151?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #152** <a href="https://app.graphite.dev/github/pr/iTwin/imodel-transformer/152?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/iTwin/imodel-transformer/152?utm_source=stack-comment).

File: packages/transformer/src/EntityUnifier.ts [hunk 1]
```diff
@@ -25,6 +26,8 @@ import {
 } from "@itwin/core-backend";
 import { Id64 } from "@itwin/core-bentley";
 
+Element;
```
Comment (MichaelBelousov) [inline packages/transformer/src/EntityUnifier.ts:29]: what is this lol

File: packages/transformer/src/test/standalone/IModelTransformerHub.test.ts [hunk 2]
```diff
@@ -72,12 +72,10 @@ import {
...
-import {
-  IModelExporter,
-  IModelImporter,
-  IModelTransformer,
-  TransformerLoggerCategory,
-} from "../../transformer";
+import { IModelExporter } from "../../IModelExporter";
+import { IModelImporter } from "../../IModelImporter";
+import { IModelTransformer } from "../../IModelTransformer";
+import { TransformerLoggerCategory } from "../../TransformerLoggerCategory";
```
Comment (MichaelBelousov) [inline packages/transformer/src/test/standalone/IModelTransformerHub.test.ts:78]: tbh not sure I like this autofix lint rule... which one is it?
(I really hope it's not the barrel one I wrote :smile:)

**Review by MichaelBelousov — COMMENTED** [review]
LGTM except fix the weird `Element;` line at the beginning

File: packages/transformer/src/test/standalone/IModelTransformerHub.test.ts (hunk 2 above)
Comment (nick4598) [reply packages/transformer/src/test/standalone/IModelTransformerHub.test.ts]: Seemed to not autofix it once I turned off the barrel one for the test dir.

File: packages/transformer/src/EntityUnifier.ts (hunk 1 above)
Comment (nick4598) [reply packages/transformer/src/EntityUnifier.ts]: i have no idea, removed it

**Review by MichaelBelousov — APPROVED** [review]

**Comment by nick4598** [comment]
### Merge activity

* **Feb 23, 1:04 PM EST**: @nick4598 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/iTwin/imodel-transformer/152).

**Pull request merged**